test/monitoring/README.md

```diff
@@ -1,25 +0,0 @@
-# localnet monitoring
```

Comment:
Do we want to preserve some of the files here (monitoring) for e2e tests? Do you think I should open an issue?

Reply:
@melekes I think we should have a `runner monitor` or `runner start --monitor` option in the future on `e2e`, so when we start an `e2e` infra you could start this monitor infra. 

 To be honest as long as we have access to these files for reference in the future I'd just get rid of them for now since it doesn't make sense to remove the `localnet` and keep these since the prometheus config is related to that. 

To be hones in `e2e` we already have an option for prometheus in the manifest so we just need to automate the monitoring infra to spin up as mentioned above with an extra subcommand or a flag to automatically start prometheus and grafana.

Reply:
@andynog I agree with that. Been wanting to do what you propose for some time.